modules/ROOT/pages/apikit-asyncapi-module-reference.adoc

```diff
@@ -0,0 +1,236 @@
+= APIkit for AsyncAPI Module Reference
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+include::reuse::partial$beta-banner.adoc[tag="apikit-asyncapi"]
+
+Learn the fundamentals of dependency, namespace, and schema information in APIkit for AsyncAPI to collaborate, build applications, and maintain codebases.
+
+== Dependency Information
+
+----
+<dependency>
+  <groupId>org.mule.modules</groupId>
+  <artifactId>mule-asynckit-module</artifactId>
+  <version>LATEST_VERSION</version> //<!--1-->
+  <classifier>mule-plugin</classifier>
+</dependency>
+----
+[calloutlist]
+.. Check https://docs.mulesoft.com/release-notes/apikit/apikit-release-notes[APIkit release notes] to get the *LATEST_VERSION*.
+
+== Namespace and Schema
+
+----
+xmlns:apikit-asyncapi="http://www.mulesoft.org/schema/mule/apikit-asyncapi" 
+xsi:schemaLocation="http://www.mulesoft.org/schema/mule/apikit-asyncapi http://www.mulesoft.org/schema/mule/apikit-asyncapi/current/mule-apikit-asyncapi.xsd">
+----
+
+== Configurations
+
+=== Config
+
+Default configuration
+
+==== Parameters
+
+[%header%autowidth.spread]
+|===
+| Name | Type | Description | Default Value | Required
+| Name | String | Name for this configuration. Connectors reference the configuration with this name | None | *Yes*
+| Api Definition | String |  API definition | None | *Yes*
```

Comment:
I think our style is to use sentence punctuation for all if anything in the column does.

Reply:
Globally.

Reply:
OK! Thanks!